app/api/api_v1/endpoints/items.py

```diff
@@ -23,20 +23,22 @@
 router = APIRouter()
 
 
-@router.post("/items/{shop_id}", response_model=ItemResponseForOwner)
+@router.post("/items/{shop_id}", response_model=ItemResponseForOwner, status_code=201)
 async def add_item_to_shop(
     *,
     shop_id: PositiveInt,
     db: Session = Depends(get_db),
     data: Item,
     current_owner: Owner = Depends(get_current_owner),
+    response: Response,
 ):
     if get_shop_by_id(db, shop_id=shop_id, owner_id=current_owner.id):
         if item_by_name_and_shop(db, shop_id=shop_id, name=data.name):
             # Item name not unique within the shop. Show exception.
             raise HTTPException(
                 status_code=409, detail="Item with this name already exists."
             )
+        response.status_code = status.HTTP_201_CREATED
```

Comment:
Is this required now after you've added `status_code` in the `post` decorator?

Reply:
Yes, this is not required.

Reply:
@Haider8 Now check it.